### PR TITLE
Bug: Fixing Image-right alignment  

### PR DESCRIPTION
**Pull request by clmedders**
## Summary

Updating placement of shortcode to address alignment issue.  Also updating partial and [Wiki](https://github.com/GSA/digitalgov.gov/wiki/Hugo-shortcodes/#images-right-aligned) for best practice to address this issue in the future.

### Preview

[Link to Preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-img-bug//2024/02/09/timeless-top-10-best-practices-for-great-government-websites/)

### Solution
Updated placement of shortcode to address error and also updated files for reference on best practice for this shortcode to avoid issues in the future

### How To Test

1. View [live site](https://digital.gov/2024/02/09/timeless-top-10-best-practices-for-great-government-websites/) and compare to [preview](https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.sites.pages.cloud.gov/preview/gsa/digitalgov.gov/cm-img-bug//2024/02/09/timeless-top-10-best-practices-for-great-government-websites/) to ensure alignment issue is address.
2. Review [Wiki](https://github.com/GSA/digitalgov.gov/wiki/Hugo-shortcodes/#images-right-aligned) to ensure update is helpful for future use.

---

### Dev Checklist

- [x] PR has correct labels
- [x] A11y testing (voice over testing, meets WCAG, run axe tools)
- [x] Code is formatted properly

### CODE REVIEW

**Review by bonnieAcameron — APPROVED** [review]

**Review by mejiaj — APPROVED** [review]
@clmedders approved, but PR description could use updates.

The preview link is broken and the wiki link sends you to the edit view.

File: themes/digital.gov/layouts/shortcodes/img-right.html [hunk 1]
```diff
@@ -1,3 +1,12 @@
+{{/* Renders image on right side of screen
+
+  ** Note for best practices, can also be found in the WIKI with short code** 
+  Short code should be added above content blocks within files to ensure alignments are correct 
+
+  <!-- TODO: move lines 11-20 into comment block for better break down of use -->
```
Comment (mejiaj) [inline themes/digital.gov/layouts/shortcodes/img-right.html:6]: This is unclear.

Comment (clmedders) [reply themes/digital.gov/layouts/shortcodes/img-right.html:6]: I have a follow up ticket to address this better and add more documentation to the file

**Review by ToniBonittoGSA — APPROVED** [review]
Approved, but Wiki will need more clarification.. added note in Trello card.

**Pull request merged**